### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by t1011**
Fixed translation into Russian

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -17,33 +17,33 @@
     <string name="lblScanExternal">Сканировать внешнее хранилище</string>
     <string name="lblNotificationMalwareDetectionTitle">Обнаружение вредоносного ПО</string>
     <string name="lblNotificationMalwareDetectionDescription">Используется для оповещения при обнаружении вредоносного ПО</string>
-    <string name="lblNotificationRealtimeTitle">Сканер в реальном времени</string>
+    <string name="lblNotificationRealtimeTitle">Онлайн-сканер</string>
```
Comment (SkewedZeppelin) [inline app/src/main/res/values-ru/strings.xml:20]: Real-time vs online?

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -17,33 +17,33 @@
...
+    <string name="main_database_updating">Обновление %s баз данных...</string>
     <string name="main_database_downloading">Загрузка %s</string>
-    <string name="main_database_download_success">Успешно загружено</string>
+    <string name="main_database_download_success">Успешно загружена</string>
     <string name="main_database_download_error">Файл не загружен, код ответа %s</string>
     <string name="main_no_database_available">База данных недоступна, сканирование не выполняется...</string>
     <string name="main_database_released_on">Выпущены %s</string>
     <string name="main_database_not_changed">Файл не изменен</string>
-    <string name="main_database_not_modified_since">от %s</string>
+    <string name="main_database_not_modified_since">%s</string>
```
Comment (SkewedZeppelin) [inline app/src/main/res/values-ru/strings.xml:34]: removed?

**Review by SkewedZeppelin — COMMENTED** [review]
Thank you for this, but can you clarify these?

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (t1011) [reply app/src/main/res/values-ru/strings.xml:20]: The line seemed too long to me and it took me a long time to figure out how to shorten the translation. Now I'm more immersed in online and real-time scanners, and I agree that it makes sense to keep the old translation. Thank you for the comment.

File: app/src/main/res/values-ru/strings.xml (hunk 2 above)
Comment (t1011) [reply app/src/main/res/values-ru/strings.xml:34]: The old version definitely does not fit here - applications in Russian are not built that way.

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (SkewedZeppelin) [reply app/src/main/res/values-ru/strings.xml:20]: Online works, I just don't want people to get the impression that files might be sent off.

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**